src/hardware/driver/dac.rs

```diff
@@ -110,7 +110,7 @@ pub mod CONFIG2 {
 /// DAC value out of bounds error.
 #[derive(Debug)]
 pub enum Error {
-    Bounds,
+    Bounds(f32, i32),
```

Comment:
Is that for debugging or do you want to generally do it like that?

Reply:
I thought there's generally no downside to returning this in the error. Ofc the error should usually not happen. But with the current sitiation that 0.0 A is a bounds error for the current sink channel it might be useful later.